test/unit/mock/config_dirs/basic/plugins.yaml

```diff
@@ -0,0 +1,6 @@
+# Plugins configuration file for InSpec
```

Comment:
Have we documented the yml behaviour? I may have missed it.

Reply:
Ahh, that file is cruft.  It's using JSON now, and is documented here - https://github.com/inspec/inspec/pull/3278/files#diff-1aac973df988b8de466d788e758d6303R35